src/swh_api_functions.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+import requests
+
+# class to genrate our own custom response
+from custom_api_response import CustomApiResponse
+
+BASE_SWH_API_URL = "https://archive.softwareheritage.org/api/1/"
```

Comment:
Will this be ever used outside of the function below?

Reply:
If by this if you mean BASE_SWH_API_URl, no it wont be used outside of any function.